### ISSUE TITLE
PHP message in group manager

### ISSUE THREAD

**Issue by DeepDiver1975** [owncloud/core#15590]
```json
{
"reqId":"1cd7dc17141019572c0f394491ab4647",
"remoteAddr":"x.x.x.x",
"app":"PHP",
"message":"array_merge(): Argument #2 is not an array at \/var\/www\/html\/owncloud\/lib\/private\/group\/manager.php#226",
"level":3,
"time":"2015-04-13T14:04:19+00:00"}

**Comment by DeepDiver1975** [comment]
core path in master already looks different - stable8 only issue

**Comment by MorrisJobke** [comment]
@DeepDiver1975 is this fixed with #15592

**Comment by DeepDiver1975** [comment]
#15592 is the fix for master - stable8 requires a different fix

**Comment by blizzz** [comment]
It can be backported quite fine actually.

**Comment by MorrisJobke** [comment]
@DeepDiver1975 @cmonteroluque the fix was also moved to 8.0.6 and therefore I will move this too.

**Comment by MorrisJobke** [comment]
Ref #17155
